babel.config.js

```diff
@@ -0,0 +1,14 @@
+module.exports = {
+  'comments': false,
+  plugins: ['source-map-support'],
```

Comment:
Was there a reason to put the other two attributes in quotes, or not putting this one in quotes? Seems like they should be consistent with each other.

Reply:
Nope, copy paste from other projects.